troubleshooting/tutorials/remote-core-dump/ecs-fargate-task-def.json

```diff
@@ -0,0 +1,64 @@
+{
+    "family": "remote-core-file-fargate",
+    "taskRoleArn": "arn:aws:iam::111111111111:role/ecs_task_role",
+    "executionRoleArn": "arn:aws:iam::111111111111:role/ecsTaskExecutionRole",
+    "networkMode": "awsvpc",
+    "cpu": "1024",
+    "memory": "2048",
+    "containerDefinitions": [{
+            "essential": true,
+            "image": "111111111111.dkr.ecr.us-west-2.amazonaws.com/aws-for-fluent-bit:debug",
```

Comment:
Should we release a debug variant in our ECR to make it easy for customers to use this?

Reply:
Yea I put this on one of our operational plans a long time ago... we have to get to it sometime...

we should though since Fluent Bit upstream has debug builds.

Reply:
Cool! yeah. seems helpful